eZ/Publish/Core/FieldType/Selection/Type.php

```diff
@@ -234,7 +234,7 @@ public function validate(FieldDefinition $fieldDefinition, SPIValue $fieldValue)
                 return array_keys($languageOptionIndexes);
             }, $fieldSettings['multilingualOptions']);
 
-            $possibleOptionIndexes = call_user_func_array('array_merge', $possibleOptionIndexesByLanguage);
+            $possibleOptionIndexes = call_user_func_array('array_merge', array_values($possibleOptionIndexesByLanguage));
```

Comment:
```suggestion
            $possibleOptionIndexes = array_merge(...$possibleOptionIndexesByLanguage);
```
...might also work. I don't remember in which version of PHP this syntax was added, but it looks like PHP7+.

Or:
```suggestion
            $possibleOptionIndexes = array_merge(...array_values($possibleOptionIndexesByLanguage));
```

Reply:
It needs to be te latter one as you cant use `...` with string keys and it leads to that same issue with php8. It seems a little bit more approachable then using `call_user_func_array` so I will go with that.